### ISSUE TITLE
1.19.3 Issue (Solved)

### ISSUE THREAD

**Issue by PaladinFallen** [2mal3/2mal3s-Tweaks#2]
The menu for the options are not working in 1.19.3

**Comment by 2mal3** [comment]
Could you give a little more context? I can't reproduce your error at the moment.

**Comment by PaladinFallen** [comment]
OKay, so first I am running it on a paper server.. Maybe I should have mentioned that first. Haha, hehe, my bad moving on. However there has been a change, I manually started the better end crystals and bush hiding and now they appear on the menu. However none of the other options are available.

![20230107202040_1](https://user-images.githubusercontent.com/43867049/211176558-be193fb3-ef1c-489d-86d8-277decc55a2b.jpg)

.......
After playing around with it, I fixed it. If anyone is having issues with the datapack not working all they need to do is uninstall it through Minecraft using the /function #2tw:uninstall then re-enabled it with /datapack enable. 


![20230107203905_1](https://user-images.githubusercontent.com/43867049/211176912-9e9dec55-eb90-47df-9b17-1680e52bf8c6.jpg)

So I can confirm with my setup it does work on a Paper Server running 1.19.3 version 371.
